tests/phpunit/elementor/includes/settings/test-settings.php

```diff
@@ -11,7 +11,6 @@ public function test_clear_css_cache_on_update_css_settings() {
 			'elementor_disable_color_schemes',
 			'elementor_disable_typography_schemes',
 			'elementor_css_print_method',
-			'elementor_optimized_dom_output',
```

Comment:
Replace with new option name.

Reply:
It's not related to the Settings class anymore

Reply:
Where you test if it clears the cache on cahnge?